README.md

```diff
@@ -1 +1,24 @@
-# coffeechats-ios
+# Pear
+
+<img width="256" alt="Pear App Icon" src="https://user-images.githubusercontent.com/29307883/100136951-afb87300-2e59-11eb-9b21-001f26e4244a.png">
+
+Coming to the App Store soon...
+
+# Build Instructions
+## Download Project
+
+0. Install [Xcode](https://itunes.apple.com/us/app/xcode/id497799835?mt=12).
+1. Clone this repository.
+2. Open Terminal and run the below code. Installing Cocoapods takes a bit, you can do Step 3 while it downloads.
+```
+cd pear-ios
+sudo gem install cocoapods // can skip if already installed
+pod install
+```
+
+3. Go to pinned messages in `#pear-ios` in Slack and download `Keys.plist` and `GoogleService-Info.plist` files. Place both files in `~/CoffeeChat/Supporting/`.
```

Comment:
I was looking at course grab's readme, and they give instructions on how to generate your own `GoogleService-Info.plist` and `Keys.plist` for those not on the team

We could prolly just paste this:

```
If you aren't an AppDev member, you can plug in your own `GoogleService-Info.plist` by generating one by following these [instructions](https://support.google.com/firebase/answer/7015592?hl=en). You will need to create a project within Firebase to this.
```
for Google and this
```
To build the project  you need a `Supporting/Keys.plist` file in the project.
```xml
?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>server-host</key>
    <string>INSERT_API_KEY</string>
    <key>google-client-id</key>
    <string>INSERT_GOOGLE_CLIENT_ID</string>
    <key>announcements-scheme</key>
    <string>INSERT_SCHEMEs</string>
    <key>announcements-host</key>
    <string>INSERT_HOST</string>
    <key>announcements-common-path</key>
    <string>INSERT_COMMON_PATH</string>
    <key>announcements-path</key>
    <string>INSERT_PATH</string>
</dict>
```
for the Keys